### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -3045,6 +3047,8 @@ emblamatic->emblematic
 eminate->emanate
 eminated->emanated
 emision->emission
+emiss->remiss, amass,
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: can also be `amiss`

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: right! btw, didn't find that the word `amissed` exists, so I didn't touch the next line

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -269,6 +269,8 @@ addtional->additional
 addtionally->additionally
 adecuate->adequate
 adequit->adequate
+adge->edge, badge,
+adges->edges, badges,
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: sorry realized one more, `adage` and `adages`

Comment (larsoner) [reply codespell_lib/data/dictionary.txt]: (probably at the end of these option lists)

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: Good catch! added and squashed commits

**Pull request merged**